File: woodroid/src/main/java/me/gilo/woodroid/data/auth/AlphabeticSorter.java
package me.gilo.woodroid.data.auth;

import org.apache.http.NameValuePair;

import java.util.Comparator;

public class AlphabeticSorter implements Comparator<NameValuePair> {

    @Override
    public int compare(NameValuePair nameValuePair1, NameValuePair nameValuePair2) {
        return nameValuePair1.getName().compareTo(nameValuePair2.getName());
    }
}

File: woodroid/src/main/java/me/gilo/woodroid/data/auth/AlphabeticSorter.kt
package me.gilo.woodroid.data.auth

import org.apache.http.NameValuePair

import java.util.Comparator

class AlphabeticSorter : Comparator<NameValuePair> {

    override fun compare(nameValuePair1: NameValuePair, nameValuePair2: NameValuePair): Int {
        return nameValuePair1.name.compareTo(nameValuePair2.name)
    }
}